doc/development_guide/sparse_backends.rst

```diff
@@ -0,0 +1,36 @@
+.. _sparse_backend:
+
+Sparse Backend Development
+==========================
+
+TensorLy's sparse support exists in the ``tensorly.contrib.sparse`` sub-package.
+This is to enable fast iteration of sparse support.  The sparse sub-package
+contains a mirror of the interfaces in the usual, dense ``tensorly`` package.
+For example, unfolding a sparse tensor would use::
+
+    >>> import numpy as np
+    >>> from tensorly.contrib.sparse import tensor, unfold
+    >>> X = tl.tensor(np.arange(24).reshape((3, 4, 2)))
+    >>> unfold(X, 0) # mode-1 unfolding
```

Comment:
This is confusing. Neither X nor unfold(X) is sparse.

Reply:
I have resolved these.